projects/admin/src/app/record/brief-view/libraries-brief-view/location/location.component.ts

```diff
@@ -0,0 +1,45 @@
+import { Component, OnInit, Input, Output } from '@angular/core';
+import { RecordUiService } from '@rero/ng-core';
+import { RecordPermissionMessageService } from 'projects/admin/src/app/service/record-permission-message.service';
+import { EventEmitter } from 'events';
+
+@Component({
+  selector: 'admin-location',
+  templateUrl: './location.component.html'
+})
+export class LocationComponent {
+
+  /** The location whose details are displayed */
+  @Input() location: any;
+
+  /** Delete location event emitter */
+  @Output() deleteLocation = new EventEmitter();
+
+  /** Constructor */
+  constructor(
+    private recordUiService: RecordUiService,
+    private recordPermissionMessage: RecordPermissionMessageService
+  ) { }
+
+  /**
+   * Delete the location
+   * @param locationPid - location PID
+   */
+  delete(locationPid: string) {
+    this.recordUiService.deleteRecord('items', locationPid).subscribe((success: any) => {
+      if (success) {
```

Comment:
When it's possible, try to type the argument (boolean).

Reply:
Done